From 7136d593ba96b64537069637c328bc65e1b66b2d Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Fri, 22 Jul 2016 18:34:39 +0000
Subject: [PATCH] get_pid_from_inode: Include limit.h

PATH_MAX and NAME_MAX are required by this file

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
Upstream-Status: Pending

 agent/mibgroup/util_funcs/get_pid_from_inode.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/agent/mibgroup/util_funcs/get_pid_from_inode.c b/agent/mibgroup/util_funcs/get_pid_from_inode.c
index 8e157ae..a494ec7 100644
--- a/agent/mibgroup/util_funcs/get_pid_from_inode.c
+++ b/agent/mibgroup/util_funcs/get_pid_from_inode.c
@@ -5,6 +5,7 @@
 #include <net-snmp/output_api.h>
 
 #include <ctype.h>
+#include <limits.h>
 #include <stdio.h>
 #if HAVE_STDLIB_H
 #include <stdlib.h>
-- 
1.8.3.1

